app/src/main/java/org/exoplatform/service/share/PostAction.java

```diff
@@ -86,6 +87,22 @@ private void setPostTextActivity() {
       postInfo.type = SocialActivity.TYPE_DEFAULT_SPACE;
   }
 
+  public static SocialRestService.PlatformInformation loadPlatformInformation() {
```

Comment:
The platform server version is already retrieved here:
https://github.com/exoplatform/exo-android/blob/fix/MOB-1875/app/src/main/java/org/exoplatform/tool/ServerUtils.java#L101

Reply:
Okay,I'll use it